chef/cookbooks/glance/recipes/ha.rb

```diff
@@ -74,4 +74,11 @@
   only_if { CrowbarPacemakerHelper.is_cluster_founder?(node) }
 end
 
+crowbar_pacemaker_order_only_existing "o-#{group_name}" do
```

Comment:
Should be `o-cl-#{group_name}` though (maybe that's what you wanted to type ;-))